app/src/main/java/ingsw/group1/findmyphone/cryptography/MessageCipher.java

```diff
@@ -0,0 +1,11 @@
+package ingsw.group1.findmyphone.cryptography;
+
+import ingsw.group1.msglibrary.Message;
+
+public interface MessageCipher<M extends Message> {
+    void setPassword(String newPassword);
```

Comment:
I understand this was in the interface I posted, and it was my mistake not to specify that it was in a "store" meaning. So setting the password to retrieve it later. Since I see the Cipher class is just an Object that does not care about persisting the password, refer to my comment below, at line 141 of `SMSCipher` and consider modifying the interface to delegate the password storing to another class. I am not God, you can and should contraddict me if I suggest something wrong